### PR TITLE
Improved build instructions for ESBMC-CHERI 

### PR DESCRIPTION
**Pull request by kunjsong01**
Added additional notes in build instructions to make it easier to follow for the user when building ESBMC-CHERI.

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -357,16 +365,20 @@ Since CHERI support is available only for a few platforms, verifying CHERI-C
 programs that use header files from the C standard library will require a setup
 of a C standard library for one of these platforms.
 
+To obtain and install a CHERI sysroot, the
+[cheribuild](https://github.com/CTSRD-CHERI/cheribuild)
+tool is the recommended way:
+```
+cheribuild.py cheribsd-mips64-hybrid disk-image-mips64-hybrid
```
Comment (fbrausse) [inline BUILDING.md]: Include `-d` and switch to the riscv64 versions.

Comment (KNG10) [reply BUILDING.md]: done. Thanks

Comment (kunjsong01) [reply BUILDING.md]: Done Thanks!

**Review by fbrausse — APPROVED** [review]
LGTM.

**Review by lucasccordeiro — APPROVED** [review]
Thanks for submitting this PR, @kunjsong01.

**Pull request merged**